composer.json

```diff
@@ -20,14 +20,17 @@
     "require": {
         "php": "^7.1",
         "ext-json": "*",
-        "becklyn/assets-bundle": "^2.2",
-        "becklyn/hosting": "^1.0",
+        "becklyn/hosting": "^1.0.2",
         "sentry/sentry-symfony": "^2.1",
         "symfony/http-kernel": "^3.4 || ^4.0"
     },
     "require-dev": {
         "twig/twig": "^2.0"
     },
+    "suggest": {
+        "becklyn/assets-bundle": "Providing the TrackJS JavaScript asset",
+        "symfony/assets": "Symfony-native fallback for apps that don't use the Becklyn Assets bundle yet to provide the TrackJS JavaScript asset"
```

Comment:
Both these description seem misleading.

Reply:
So what would you recommend then? `Needed to provide URL to embedded javascript asset` or something like that?

Reply:
```
"becklyn/assets-bundle": "For embedding the frontend monitoring using Becklyn's asset() twig function.",
"symfony/assets": "For embedding the frontend monitoring using Symfony's asset() twig function.",
```